bugbot/rules/component.py

```diff
@@ -76,6 +81,9 @@ def get_bz_params(self, date):
             "f9": "CP",
         }
 
+    def meets_threshold(self, bug_data, threshold):
+        return bug_data["prob"][bug_data["index"]] >= threshold
+
```

Comment:
We might have it as a local function instead. The function should also select the threshold. We could add docstrings to clarify the scope if necessary.

Reply:
Done here: ff5b63363ce48bf29f2ba730efd91ff7de42a0d5